contracts/optimistic-ethereum/OVM/precompiles/OVM_ProxySequencerEntrypoint.sol

```diff
@@ -4,7 +4,13 @@ pragma solidity ^0.7.0;
 import { Lib_SafeExecutionManagerWrapper } from "../../libraries/wrappers/Lib_SafeExecutionManagerWrapper.sol";
 
 /**
- * @title OVM_ProxySequencerEntrypoint
+ * @title OVM_ProxySequencerEntrypoint 
+ * @dev The Proxy Sequencer Entrypoint is a predeployed proxy to the implementation of the 
+ * Sequencer Entrypoint. This will enable the Optimism team to upgrade the Sequencer Entrypoint 
+ * contract.
```

Comment:
could be worth noting here that this is not an admin key, since it has no safety/authentication logic -- worse we could do is change it to something that doesn't correctly supply inputs to the contract wallets.